app/themes/ukis/templates/introduction.html

```diff
@@ -0,0 +1,46 @@
+{% extends 'introduction.html' %}
+
+{% block header %}
+  {% include 'partials/header.html' %}
+  {% with is_hero = true %}
+    {% include theme('partials/topbar.html') %}
+  {% endwith %}
+{% endblock %}
+
+{% block business_details %}
+  <div class="u-mb-xs">
+    {% if meta.survey.region_code == 'GB-NI' %}
+      <img src="{{ theme_static('logo-dof.png') }}" alt="Department of Finance" width="183" />
+    {% else %}
+      <img src="{{ theme_static('logo-beis.png') }}" alt="Department for Business, Energy and Industrial Strategy" width="660" />
+    {% endif %}
+  </div>
+
+  {{super()}}
```

Comment:
@ajmaddaford That is fixed now, just using the `legal_basis` hook to toggle the two bits of text. Not sure if the template is the best place to define this wording though.